nav2_bt_navigator/behavior_trees/parallel.xml

```diff
@@ -18,12 +18,12 @@
 <root main_tree_to_execute="MainTree">
   <BehaviorTree ID="MainTree">
     <SequenceStar name="root">
-      <ComputePathToPose endpoints="${endpoints}" path="${path}"/>
+      <ComputePathToPose goal="${goal}" path="${path}"/>
```

Comment:
Just the goal is now sent to the global planner. The global planner will get the robot's current location. For this release, the tolerance is a parameter for the global planner. In the future, we'll create commands at the top level that communicate the quality with which the operation should be performed.